### ISSUE TITLE
Fulfill the original vision 7 years ago

### ISSUE THREAD

**Issue by neolee** [LightTable/townhall#3]
https://vimeo.com/40281991

**Comment by JJ-Atkinson** [comment]
I want to help out with the code blocks feature. Currently I'm near finals, but I'll be able to throw some time at it over the summer. Should it be a plugin or built-in? Also, when should everything be stable enough for me to build on? I saw that LT is being updated for the new versions for CLJS and Electron.

**Comment by Mouvedia** [comment]
If you wanna help join https://gitter.im/LightTable/LightTable and discuss it with @prertik.

**Comment by gabrielbusta** [comment]
The discussion moved here https://lighttable.discourse.group/

**Comment by bmix** [comment]
And it's gone: 

> Sorry, the site you are looking for does not exist.

**Comment by prertik** [comment]
@bmix Yeah, not many people moved there and due to stall in activity there, I think it's closed. Will start discord in the beginning of october.
